### DISCUSSION TITLE
Filter already set tags from tag list

### DISCUSSION THREAD

**Discussion by rYR79435** [mateusz-bak/openreads#712] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I assign a tag to a book, a list of existing tags is displayed. That list contains tags which have already been assigned to that book. Given that selecting a tag a second time doesn't add it to the book a second time, it is redundant to display it as a choice.

**Describe the solution you'd like**
When assigning tags to a book, remove already assigned tags from the list of all existing tags.

**Comment by mateusz-bak** [comment]
Are you able to handle this @apobrt ?

**Comment by th8m0z** [comment]
I can also handle this

**Comment by mateusz-bak** [comment]
Alright, check with @apobrt then.

**Comment by th8m0z** [comment]
Right now, when testing on the Android emulator, the tags feature seems to be broken. 
The addNewTag function throws because the tag is empty. 

I'm assuming the expected behaviour would be, that the user can add their own tags?

**Comment by th8m0z** [comment]
Here is a short video displaying the problem:


https://github.com/mateusz-bak/openreads-android/assets/41947147/b74033ed-4652-4b8a-a93c-523b65fda2d1

**Comment by mateusz-bak** [comment]
There was a new controller that was overwriting original tags controller. I removed it and now adding tags works, you can rebase. https://github.com/mateusz-bak/openreads-android/pull/430

**Comment by mateusz-bak** [comment]
@th8m0z how is the work going?

**Comment by th8m0z** [comment]
Should be done in the next couple of days! @mateusz-bak

**Comment by onbreadassist** [comment]
Is the issue fixed ?

**Comment by nikkhilpareek** [comment]
is this issue fixed or needs to be fixed?

**Comment by mateusz-bak** [comment]
Issue still persists
